### DISCUSSION TITLE
wallabag.org seems to have been hijacked or broken

### DISCUSSION THREAD

**Discussion by ghost** [wallabag/wallabag#6548] [General]
The Wallabag logo has been replaced with something called "boxify":
![image](https://github.com/wallabag/wallabag/assets/56515875/bbd6c72d-8450-426a-a5a2-5dd932777ac3)

...but the Wallabag name is still present in the text:
![image](https://github.com/wallabag/wallabag/assets/56515875/7b2528cd-19e1-43fd-ad86-4017833b9de5)

The formatting of the site is broken:
![image](https://github.com/wallabag/wallabag/assets/56515875/d1c30f6d-aa04-45b8-b5ee-f973c12674a1)
![image](https://github.com/wallabag/wallabag/assets/56515875/7d6cfb3a-682b-4328-846f-5ac58fc7a88b)

And the footer looks like this:
![image](https://github.com/wallabag/wallabag/assets/56515875/7a3f6b5e-7a7e-4780-a658-da0efd029809)

**Comment by j0k3r** [comment]
@nicosomb updated it few days ago. Maybe it's related?

**Reply by nicosomb** [reply]
Yes. Sorry I had some bugs with the update.

**Comment by nicosomb** [comment]
Fixed (in progress). 

We now have a repository for the website https://github.com/wallabag/website
